zeth_contracts/package.json

```diff
@@ -10,16 +10,16 @@
     "deploy": "truffle deploy",
     "test": "truffle test",
     "debug": "truffle debug",
-    "check": "node_modules/solium/bin/solium.js --dir contracts"
+    "check": "node_modules/solhint/solhint.js 'contracts/**/*.sol'"
```

Comment:
This command does not exit with error for warnings (I tried changing a constant to lower case).  This means that the CI will never detect these issues.